status/statistics.py

```diff
@@ -22,19 +23,55 @@ def get_stats_data(db,view,gl=None):
             data[row.key]=row.value
     return data
 
+def clean_application_keys_y(raw_data):
+    # Normalise Application names to approved list
+    # Don't let Denis see this, it will make him cry
+    with open('application_keys.yaml') as f:
+        app_cats=yaml.load(f)
+    data={}
+    for year in raw_data:
+        data[year] = {}
+        for c in raw_data[year]:
+            try:
+                data[year][app_cats[c]] += raw_data[year][c]
+            except KeyError:
+                try:
+                    data[year][app_cats[c]] = raw_data[year][c]
+                except KeyError:
+                    data[year][c] = raw_data[year][c]
+    return data
+
+def clean_application_keys(raw_data):
```

Comment:
Yeah I thought of that. But then I have to call the function in two different ways which I'd rather avoid. I did something similar but using an `assert` statement to figure it out automatically in cd0a37c8770bc753d813ec5cabd978828e171866